lombok-plugin/src/main/java/io/freefair/gradle/plugins/lombok/LombokPlugin.java

```diff
@@ -89,7 +89,14 @@ private void configureJavaPluginDefaults() {
 
         project.getPlugins().withType(JacocoPlugin.class, jacocoPlugin -> configureForJacoco());
 
-        project.getPlugins().withId("com.github.spotbugs", spotBugsPlugin -> configureForSpotbugs(javaPluginConvention));
+        if (isSpotbugsConfigurationNeeded()) {
+            configureForSpotbugs(javaPluginConvention);
+        }
+    }
+
+    private boolean isSpotbugsConfigurationNeeded() {
+        return project.getPlugins().hasPlugin("com.github.spotbugs")
```

Comment:
This will only work, if the spotbugs or sonatype plugin are applied before the lombok plugin.
If the lombok plugin is applied first, this will return false and nothing will happen.

Reply:
You are right. Changed!